zipline/finance/trading.py

```diff
@@ -78,7 +78,7 @@ class TradingEnvironment(object):
     def __init__(
         self,
         load=None,
-        bm_symbol='^GSPC',
+        bm_symbol='SPY',
```

Comment:
Yes. We should do that. We should also remove data in the `example_data` tarball for older versions of pandas.